train_instruct_pix2pix_sdxl.py

```diff
@@ -868,24 +950,15 @@ def compute_time_ids(batch_size):
 
     for epoch in range(first_epoch, args.num_train_epochs):
         unet.train()
-        train_loss = 0.0
         for step, batch in enumerate(train_dataloader):
-            # Skip steps until we reach the resumed step
-            if (
-                args.resume_from_checkpoint
-                and epoch == first_epoch
-                and step < resume_step
-            ):
-                if step % args.gradient_accumulation_steps == 0:
-                    progress_bar.update(1)
-                continue
-
```

Comment:
This is not required with `wds`, as with web dataset we do random resuming, which with large datasets performs similar to exact resuming. cf https://github.com/webdataset/webdataset/issues/177#issuecomment-1118186022